oxide/types.go

```diff
@@ -3295,15 +3297,25 @@ type InstanceSerialConsoleData struct {
 type InstanceState string
 
 // InstanceUpdate is parameters of an `Instance` that can be reconfigured after creation.
+//
+// Required fields:
+// - Memory
+// - Ncpus
 type InstanceUpdate struct {
-	// AutoRestartPolicy is the auto-restart policy for this instance.
+	// AutoRestartPolicy is sets the auto-restart policy for this instance.
 	//
-	// If not provided, unset the instance's auto-restart policy.
+	// This policy determines whether the instance should be automatically restarted by the control plane on failure. If this is `null`, any explicitly configured auto-restart policy will be unset, and the control plane will select the default policy when determining whether the instance can be automatically restarted.
```

Comment:
Not a problem to fix in this PR, but these very long single-line comments do not look good on pkg.go.dev, see https://pkg.go.dev/github.com/oxidecomputer/oxide.go@v0.1.0-beta9/oxide#InternetGatewayCreate as an example.

I've opened https://github.com/oxidecomputer/oxide.go/issues/248 for tracking.

Reply:
Yeah, that's a good idea. Thanks for opening up the issue!